SnapWar/Entities/SWUserEntity.swift

```diff
@@ -10,14 +12,36 @@ class SWUserEntity {
     
     private init() {
         self.isUserLoggedIn = false
+        self.fromDefaults()
     }
     
+    fileprivate func fromDefaults() {
+        let defaults = UserDefaults.standard
+        if let externalId = defaults.object(forKey: "externalId") as? String {
+            self.externalId = externalId
+        }
+        if let displayName = defaults.object(forKey: "displayName") as? String {
+            self.displayName = displayName
+        }
+        if let avatarURL = defaults.object(forKey: "avatarURL") as? String {
+            self.avatarURL = avatarURL
+        }
+        
+        if(externalId != nil && displayName != nil && avatarURL != nil) {
```

Comment:
im fine with this for now! when we integrate with our server this logic will change a bit